Source: written by rousseau, from Debian
Upstream: yes, Debian bug #626201, http://svn.debian.org/wsvn/pcsclite/?sc=1&rev=5732
Reason: Fix a link failure if --disable-usb is used.
--- trunk/PCSC/src/winscard_svc.c (original)
+++ trunk/PCSC/src/winscard_svc.c Wed May 11 06:52:39 2011
@@ -352,8 +352,10 @@
 			{
 				/* nothing to read */
 
+#ifdef USE_USB
 				/* wait until all readers are ready */
 				RFWaitForReaderInit();
+#endif
 
 				/* dump the readers state */
 				ret = MessageSend(readerStates, sizeof(readerStates), filedes);
